config/environments/development.rb

```diff
@@ -28,8 +32,14 @@
   # Do not compress assets
   config.assets.compress = false
 
+  # Generate digests for assets URLs.
+  #
+  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
+  # yet still be able to expire them through the digest params.
+  config.assets.digest = !!ENV["PROFILE"]
+
   # Expands the lines which load the assets
-  config.assets.debug = false
+  config.assets.debug = !!ENV["PROFILE"]
```

Comment:
I gave this PR a try but maybe this settings is messing it up.
I dont get the assets bundled :man_shrugging: 
thsi is not production like:
![image](https://user-images.githubusercontent.com/1640378/79640374-381abb00-8189-11ea-9674-87850b457b0c.png)